pydriller/git_repository.py

```diff
@@ -265,3 +265,26 @@ def _useless_line(self, line: str):
         # this covers comments in Java and Python, as well as empty lines. More have to be added!
         return not line or line.startswith('//') or line.startswith('#') or line.startswith("/*") or \
                line.startswith("'''") or line.startswith('"""') or line.startswith("*")
+
+    def get_commits_modified_file(self, filepath: str) -> List[Commit]:
+        all_commits = self.get_list_commits()
+
+        dict_commits = {}
+        for commit in all_commits:
+            dict_commits[commit.hash] = commit
+
+        # getting the absolute path of the file
+        path = str(Path(filepath).absolute())
```

Comment:
There is an error if using `.absolute()`, probably depending on the machine. Running on my Mac OS, the test fails. If you remove the `.absolute()`, everything works.

Reply:
Interesting....then leave it like this, I will investigate personally after merging the PR. Thanks for letting me know this!